src/lib/ngsi/Scope.cpp

```diff
@@ -308,6 +306,46 @@ int Scope::fill
     return -1;
   }
 
+  //pointVectorRelease(pointV); ??
```

Comment:
Not sure if the pointV.clear() and pointVectorRelease(pointV) statements are ok along all the fill() method. @kzangeli , as original author of the fill() method, could you have a look to be sure, so we can early remove any potential leak?